src/main/java/pl/touk/sputnik/connector/http/HttpConnector.java

```diff
@@ -59,4 +63,8 @@ public String consumeAndLogEntity(@NotNull CloseableHttpResponse response) throw
         log.info("Entity {}: {}", REQUEST_COUNTER, content);
         return content;
     }
+
```

Comment:
[Checkstyle] ERROR: Variable 'REQUEST_COUNTER' explicitly initialized to '0' (default value for its type).